piIOComm.c

```diff
@@ -413,7 +413,7 @@ int piIoComm_init(void)
 void piIoComm_finish(void)
 {
 	if (piIoComm_fd_m != NULL) {
-		pr_info_serial("filp_close %d\n", (int)piIoComm_fd_m);
+		pr_info_serial("filp_close %zd\n", (size_t)piIoComm_fd_m);
```

Comment:
https://www.kernel.org/doc/html/latest/core-api/printk-formats.html#pointer-types

Reply:
Other question @l1k and others: What is the purpose of this debug message? Is it probably better to just remove it if it is not used? The origin is a commit from MD 5 years ago and personally I'm not able to see the purpose of this kind of information...

Reply:
See my comment for the "file open" case.

Reply:
Done in 3d29666395653a6d2bc54a6b38e4fb8125a42106